guides/v2.2/config-guide/cli/config-cli-subcommands-cron.md

```diff
@@ -48,6 +48,22 @@ Command options:
 
 where `--group` specifies the cron group to run (omit this option to run cron for all groups)
 
+For run indexing cron job,
+
+`php bin/magento cron:run --group index`
+
+Output will be display as,
```

Comment:
Delete this line and `Ran jobs by schedule`. It's just not an insightful return message.